### PR TITLE
Confirm page container created

### PR DESCRIPTION
**Pull request by OlaShabalina**
## Created a container component which accepts the following props:
* title
* icon
* describtion
* any content as children
image

### CODE REVIEW

**Comment by OlaShabalina** [comment]
Hi team, don't get confused if you see changes to DaysOfTheWeek components here, it's my pull request from CodeForAustralia:main to sync the code I had. 😄

File: pages/form/ConfirmWFH.jsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+import LinkButton, { BackButton } from "../../components/LinkButton/LinkButton";
+import { useRouter } from "next/router";
+import { sendLogs } from "../../utils/sendLogs";
+
+import ConfirmDetailsContainer from "../../components/ConfirmDetailsContainer/ConfirmDetailsContainer";
+
+/* for now, we pass hard-coded data from the porent component (icon and title), 
+eventually will render based on what user selects during previous steps:
+*/
+import House from "../../public/images/other/House.svg";
```
Comment (hqtan) [inline pages/form/ConfirmWFH.jsx:20]: just a small typo here (was causing a build error with Vercel deployment)
 
```suggestion
import House from "../../public/images/other/house.svg";
```

**Review by hqtan — CHANGES_REQUESTED** [review]
Hi @OlaShabalina :wave: 

Thx again for your PR :bow: 

so far only spotted a small typo (reported as a Vercel build error); 
Great work on PR! :+1:

File: pages/form/ConfirmWFH.jsx (hunk 1 above)
Comment (OlaShabalina) [reply pages/form/ConfirmWFH.jsx]: good pickup @hqtan , I changed casing for all icons I added and associated paths 👍

File: components/ConfirmDetailsContainer/ConfirmDetailsContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import { Children, useState } from "react";
+import { Text, Box, Flex, Collapse, Spacer, Icon, Button } from "@chakra-ui/react";
+import Pencil from "../../public/images/other/pencil.svg"
+
+export default function ConfirmDetailsContainer({ methodIcon, title, children, describtion }) {
```
Comment (hqtan) [inline components/ConfirmDetailsContainer/ConfirmDetailsContainer.jsx:5]: just a small typo with for one of the arguments

```suggestion
export default function ConfirmDetailsContainer({ methodIcon, title, children, description }) {
```

File: components/ConfirmDetailsContainer/ConfirmDetailsContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          pl="15px"
+        >
+          <Text
+            color="#044B7F"
+            fontWeight="500"
+            fontSize="24px"
+          >
+            {title}
+          </Text>
+          { describtion }
```
Comment (hqtan) [inline components/ConfirmDetailsContainer/ConfirmDetailsContainer.jsx:46]: you forgot about updating this one too @OlaShabalina !

```suggestion
          { description }
```

File: components/ConfirmDetailsContainer/ConfirmDetailsContainer.jsx (hunk 2 above)
Comment (OlaShabalina) [reply components/ConfirmDetailsContainer/ConfirmDetailsContainer.jsx]: good pick-up, just fixed it in all 3 places 👍

File: components/ConfirmDetailsContainer/ConfirmDetailsContainer.jsx (hunk 3 above)
Comment (hqtan) [reply components/ConfirmDetailsContainer/ConfirmDetailsContainer.jsx:46]: I'll just commit this for you, Olga :+1:

**Review by hqtan — APPROVED** [review]
Thx for the work with this component @OlaShabalina !
Gonna approve this so we can test out this component :+1: 
sorry for the wait! :bow: 

lgtm

**Pull request merged**